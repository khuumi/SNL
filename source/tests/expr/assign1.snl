var is 6.3 - 9 * 1.45