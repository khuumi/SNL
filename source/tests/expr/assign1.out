Assign var (Binop (Float 6.3) Sub (Binop (Int 9) Mult (Float 1.45)))